website/content/en/docs/helm/reference/advanced_features.md

```diff
@@ -110,3 +112,9 @@ spec:
   service:
     port: 8080
 ```
+
+Setting this annotation to `True` and updating the spec (for example changing to `spec.replicaCount:3`) will cause this CR to be reconciled and upgraded with the `force` option. This can be verified in the log message when an upgrade succeeds: 
```

Comment:
```suggestion
Setting this annotation to `True` and updating the spec (for example changing to `spec.replicaCount: 3`) will cause this CR to be reconciled and upgraded with the `force` option. This can be verified in the log message when an upgrade succeeds: 
```